src/AudioUnit/AUEnums.cs

```diff
@@ -189,12 +189,20 @@ public enum AudioObjectPropertySelector : uint
 		TranslateUIDToBox = 1969841250, // 'uidb'
 		ClockDeviceList = 1668049699, //'clk#'
 		TranslateUidToClockDevice = 1969841251, // 'uidc',
+#if XAMCORE_3_0
```

Comment:
The defines in this enum are somewhat confusing.

I think this would simplify it:

```cs
[NoTV][NoWatch]
#if NET
[NoiOS]
#endif
public enum AudioObjectPropertySelector : uint
{
...
#if !NET
 		[Deprecated (PlatformName.iOS, 15,0, message : "Use the 'ProcessIsMain' element instead.")]
#endif
 		[Deprecated (PlatformName.MacCatalyst, 15,0, message : "Use the 'ProcessIsMain' element instead.")]
 		[Deprecated (PlatformName.MacOSX, 12,0, message : "Use the 'ProcessIsMain' element instead.")]
 		[Obsolete ("Use the 'ProcessIsMain' element instead.")]
 		ProcessIsMaster = 1835103092, // 'mast'
 		[MacCatalyst (15,0), Mac (12,0)]
 		ProcessIsMain = 1835100526, // 'main'
...
 		[MacCatalyst (15,0), Mac (12,0)]
 		ProcessMute = 1634758765, // 'appm'
}
```

That said, the changes here look correct, so this could go in another PR.

Reply:
Like it, let's save it for a follow up PR.